modal_deployment.py

```
@@ -1,11 +1,13 @@
 import modal
 import os
 import logging
+import structlog
+from cognee.shared.logging_utils import setup_logging
 import asyncio
 import cognee
 import signal
 
-from cognee.shared.utils import setup_logging
+from cognee.shared.logging_utils import setup_logging
```

Ruff (0.8.2)

Redefinition of unused `setup_logging` from line 5

Remove definition: `setup_logging`

(F811)